_raml/apps/products-categories/products-categories.raml

```diff
@@ -20,5 +20,12 @@ post:
 delete:
   description: Deletes one or more products from a category.
   securedBy: [products_write]
+  queryParameters:
+    categoryId:
+      description: Filters products by category ID.
```

Comment:
description: The product which should be removed from the specified categories (can be used multiple times)